public/i18n/qqq.json

```diff
@@ -1,3 +1,16 @@
 {
-  "query-builder-run-query": "Text on the button that runs the query"
+  "query-builder-heading": "Title of the page",
+  "query-builder-description": "A short description on what query builder is",
+  "query-builder-find-all-items": "header for part of the form finding all items",
+  "query-builder-run-query": "Text on the button that runs the query",
+  "query-builder-input-value": "Text of value of the property in the form",
+  "query-builder-input-enter-value": "Placeholder of the value input",
+  "query-builder-property-lookup-enter-property": "Label of the property input",
+  "query-builder-property-lookup-property": "Placeholder of the property input",
```

Comment:
The `query-builder-property-lookup-property` and `query-builder-property-lookup-enter-property` are mixed up. It would probably be better if the message names could explicitly include their purpose. So calling them something like `query-builder-property-lookup-label` and `query-builder-property-lookup-placeholder`

Reply:
You're right. Done now.

Reply:
:+1: